index.html

```diff
@@ -205,18 +216,20 @@ <h2>Languajes</h2>
         <i class="fa-solid fa-angle-right"></i>
         </button>
     </div>
-    <div class="skills">
-        Skills
-        <button class="right2">
-        <i class="fa-solid fa-angle-right"></i>
-        </button>
+        <div class="skills">
+            Skills
+            <button class="right2">
+                <i class="fa-solid fa-angle-right"></i>
+            </button>
+        </div>   
     </div>
 </section>
 
+
 <section class="form-section">
     <div class="form">
         <h2 class="form-title-2">Contact me</h2>
-        <p>If you have an application you are interested in developing, a feature 
+        <p id="pa">If you have an application you are interested in developing, a feature 
             that you need built or a project that needs coding. I’d love to help with it </p>
         <form class="form-media">
```

Comment:
- [ ] The buttton `            <button type="submit" class="b1">Get It Touch</button>` should be centred, kindly center the button and make sure the padding is semilare too the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_093426](https://user-images.githubusercontent.com/25227022/173180808-6573bad9-bf94-416f-a664-686689676214.png)|![Screenshot_20220611_093411](https://user-images.githubusercontent.com/25227022/173180815-bf60d93d-127b-43b0-8821-ca6e3693ab8c.png)|